docs/reference/search/search-your-data/retrieve-selected-fields.asciidoc

```diff
@@ -388,7 +388,7 @@ of the above metadata fields, this field will also be retrievable using either
 the alias name or via wildcard patterns that match the alias.
 
 [discrete]
-[[Ignored-field values]]
+[[ignored-field-values]]
 ==== Ignored field values
 The `fields` section of the response only returns values that were valid when indexed.
 If your search request asks for values from a field that ignored certain
```

Comment:
```suggestion
If your search request asks for values from a field that ignored certain values 
```

Reply:
I think that there's a missing word here, which makes the sentence ready awkwardly. I know that it's not part of your changes, but would be good to fix.